frontend/src/components/student/NavButtons.tsx

```diff
@@ -37,7 +40,7 @@ const NavButtons = (): React.ReactElement => {
       )}
       <Spacer />
       {isLastQuestion ? (
-        <SubmitButton />
+        <SubmitButton isDisabled={isPreviewMode} />
```

Comment:
I'm not sure if the best thing to do here is to disable the submit button? Open to thoughts.

Reply:
I think disabling the button makes a lot of sense!